tests/testthat/test-util.R

```diff
@@ -20,3 +20,17 @@ test_that("path encoding", {
   expect_equal(decode_path(c("foo:bar", "foo:bar:baz")),
                c("foo/bar", "foo/bar/baz"))
 })
+
+test_that("clean input text", {
+  ## For VIMC-1018
+  expect_equal(clean_input_text('"foo"'), "foo") # strip quotes
+  expect_equal(clean_input_text('  foo'), "foo") # strip leading whitespace
+  expect_equal(clean_input_text('foo  '), "foo") # strip trailing whitespace
+  expect_equal(clean_input_text(' foo '), "foo") # strip all whitespace
+  expect_equal(clean_input_text(' "foo" '), "foo") # strip all whitespace/quotes
+  expect_equal(clean_input_text(" 'foo' "), "foo") # strip all whitespace/quotes
+  ## But allow use of quotes to preserve whitespace
+  expect_equal(clean_input_text(' " foo " '), " foo ")
+  expect_equal(clean_input_text(" ' foo ' "), " foo ")
+  expect_equal(clean_input_text("f oo"), "f oo")
```

Comment:
Could consider adding these two tests

expect_equal(clean_input_text('"   "'),"   ")
expect_equal(clean_input_text(clean_input_text('"   "')),"")

to show the (useless) special case that only-whitespace contained within quotes is preserved.